### PR TITLE
Fix getAccessToken in ui-test-app

### PR DESCRIPTION
**Pull request by kabentley**
@NancyMcCallB Can you try out this change?

### CODE REVIEW

File: test-apps/ui-test-app/src/frontend/appui/imodelopen/IModelOpen.tsx [hunk 1]
```diff
@@ -67,14 +67,11 @@ export class IModelOpen extends React.Component<IModelOpenProps, IModelOpenState
       });
     }
 
-    if (undefined === this.props.getAccessToken)
-      return;
-
     const token = await this.props.getAccessToken();
-    if (undefined === token)
```
Comment (kabentley) [inline test-apps/ui-test-app/src/frontend/appui/imodelopen/IModelOpen.tsx:74]: @NancyMcCallB when I ran ui-test-app I got an error that the token was blank. I think this is why, but now I just get "fetching project information" spinner and i'm not sure how to debug.

File: core/frontend/src/tile/RealityModelTileTree.ts [hunk 2]
```diff
@@ -838,18 +838,6 @@ export class RealityModelTileClient {
       this.rdsProps.projectId = iTwinId;
   }
 
-  private async getAccessToken(): Promise<AccessToken | undefined> {
-    if (!IModelApp.authorizationClient)
-      return undefined;
-    let accessToken: AccessToken | undefined;
-    try {
-      accessToken = await IModelApp.authorizationClient.getAccessToken();
```
Comment (kabentley) [inline core/frontend/src/tile/RealityModelTileTree.ts:846]: @pmconne this is a similar problem. Maybe that's what you're seeing with reality data?

**Comment by NancyMcCallB** [comment]
Looking at it now.

**Review by NancyMcCallB — APPROVED** [review]

File: core/frontend/src/tile/RealityModelTileTree.ts (hunk 2 above)
Comment (pmconne) [reply core/frontend/src/tile/RealityModelTileTree.ts:846]: Nope, still getting CORS errors.

Comment (kabentley) [reply core/frontend/src/tile/RealityModelTileTree.ts:846]: sad face emoji

File: test-apps/ui-test-app/src/frontend/appui/imodelopen/IModelOpen.tsx (hunk 1 above)
Comment (kabentley) [reply test-apps/ui-test-app/src/frontend/appui/imodelopen/IModelOpen.tsx:74]: @NancyMcCallB is ui-test-app working correctly for you now?

**Review by MarcBedard8 — APPROVED** [review]
Change looks good, but I'll still getting an empty token in RealityModelTileTree:getRootDocument. Might be cause by a bad configuration on my side.

**Review by pmconne — APPROVED** [review]

**Pull request merged**